src/main/java/net/minecraftforge/event/entity/living/AgeableEntityGrowingAdultEvent.java

```diff
@@ -0,0 +1,28 @@
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.entity.AgeableEntity;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.ForgeEventFactory;
+
+/**
+ * This event is fired when an {@link AgeableEntity} becomes an adult. <br>
+ * It is fired via {@link ForgeEventFactory#onAgeableEntityGrowingAdult(AgeableEntity)}.
+ * <br>
+ * This event not is {@link net.minecraftforge.eventbus.api.Cancelable}.
+ * <br>
+ * This event does not have a result. {@link HasResult} <br>
+ * <br>
+ * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+ */
+public class AgeableEntityGrowingAdultEvent extends LivingEvent
```

Comment:
This actually isn't fired when a entity becomes an adult.
It's fired when it's age changes. So it COULD be fired when an entity turns into a child FROM an adult.
Also the name needs work to reflect that.

Reply:
Renamed event to reflect that it can be child->adult or vice-versa, and updated header comments. Now called `AgeableEntityAgeChange`.